.storybook/manager.js

```diff
@@ -1,6 +1,6 @@
 import { addons } from '@storybook/addons';
 import { create } from '@storybook/theming/create';
-import brandImage from '../assets/images/gravitee-logo-inline.png';
+import brandImage from '../assets/images/gravitee-logo-dark-blue.svg';
```

Comment:
why not keeping the same generic name ?

Reply:
Because we now have the logo in 3 different colours, 2 different shapes and according to the place it is used we need one or another.